src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  const getSum = (a) => {
+    count++;
+
+    return (b) => {
+      if (count > 2 && count % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return a + b;
+      }
```

Comment:
```suggestion
      if (count > 2 && count % 2 === 0) {
        return 'Bzzz... Error!';
      } 
      
      return a + b;
```
try to avoid else where it is possible